src/main/java/com/easypost/http/Requestor.java

```diff
@@ -81,10 +83,6 @@ static Map<String, String> generateHeaders(String apiKey) throws MissingParamete
                 convertSpaceToHyphen(System.getProperty("os.arch")),
                 convertSpaceToHyphen(System.getProperties().getProperty("java.vm.name"))));
 
-        if (apiKey == null || apiKey.isEmpty()) {
```

Comment:
# Reviewer note:

This is no longer needed since there is a check statement in the constructor when initializing the `EasyPostClient` object